addins/CFSample/README.md

```diff
@@ -0,0 +1,12 @@
+# License
+See https://officedev.github.io/custom-functions/LICENSE
+
+# Purpose
+This add-in is for internal testing purposes.It is used to test CustomFunction which contains different sample of Functions like: DialogApi, OfficeStorage, streaming functions etc, along with a Taskpane which contain OfficeStorage API to get/set Token from the Storage. This Addin basically runs CustomFunctions inside the JSRuntime.
```

Comment:
I suggest removing the last sentence altogether - it sounds too casual, it doesn't convey any new information, and most importantly there is no such concept as 'JSRuntime' none-the-less public.